Makefile

```diff
@@ -75,11 +75,19 @@ routereflector.tar:
 	docker pull calico/routereflector:latest
 	docker save --output routereflector.tar calico/routereflector:latest
 
-## Download the latest docker binary
 docker:
+	# Download the docker 1.9.1 binary
 	curl https://get.docker.com/builds/Linux/x86_64/docker-1.9.1 -o docker
 	chmod +x docker
 
+docker-1-10:
```

Comment:
This all seems rather specific to the semaphore environment, I would just add it there.  Does anything rely on the docker make target - seems odd that we have it now that there are standard ways to install 1.9.1